libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.scss

```diff
@@ -73,10 +68,5 @@ watt-nav-list {
     .mat-expansion-panel-body {
       padding: 0;
     }
-
-    .mat-nav-list .mat-list-item .mat-list-item-content {
-      padding: var(--watt-space-s) var(--watt-space-m) var(--watt-space-s)
-        var(--watt-space-l);
-    }
```

Comment:
issue: the nested nav list had a double padding before. It's not there anymore.

Before
![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/2e2e2d53-6277-4609-8ced-49aae61d5c83)

After
![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/887a0029-459b-416a-833e-67b839db13e7)

Reply:
Good catch!, fix now